skyportal/handlers/api/candidate.py

```diff
@@ -439,6 +442,7 @@ def get(self, obj_id=None):
 
         page_number = self.get_query_argument("pageNumber", None) or 1
         n_per_page = self.get_query_argument("numPerPage", None) or 25
+        query_id = self.get_query_argument("queryID", None)
```

Comment:
I've intentionally not documented this, as it is only for front-end usage and could likely confuse users looking through the API docs.

Reply:
Maybe put a quick inline comment here saying that's why it's not in the API docs? In case we forget in the future.